.devcontainer.json

```diff
@@ -56,7 +56,7 @@
   // },
 
   // Use 'postCreateCommand' to run commands after the container is created.
-  // "postCreateCommand": "yarn install"
+  "postCreateCommand": "curl -fsSL https://aka.ms/install-azd.sh | bash"
```

Comment:
Why not use the azd feature? That will take care of architecture issues so that it works on ARM machines.

Reply:
That is true, there are issues with Macs with Apple Silicon. But how to get azd installed in the first place? How do we solve that? @pamelafox thx.

Reply:
Use the feature: https://github.com/Azure-Samples/azure-search-openai-demo/blob/e91496076f2b061b0d9f898d3bdc3b9b87633323/.devcontainer/devcontainer.json#L11